nupic/research/frameworks/pytorch/imagenet/imagenet_tune.py

```diff
@@ -125,7 +126,10 @@ def _train(self):
             # Return the results from the first remote function
```

Comment:
This comment is no longer correct. It looks like `aggregate_eval_results` is averaging the results